docs/AccessScalarProducts.md

```diff
@@ -0,0 +1,178 @@
+scalardl-auditor-metrics         ClusterIP      10.105.119.158   <none>        8080/TCP                          99s
+scalardl-ledger-envoy            LoadBalancer   10.96.239.167    <pending>     50051:32714/TCP,50052:30857/TCP   109s
+scalardl-ledger-envoy-metrics    ClusterIP      10.97.204.18     <none>        9001/TCP                          109s
+scalardl-ledger-headless         ClusterIP      None             <none>        50051/TCP,50053/TCP,50052/TCP     109s
+scalardl-ledger-metrics          ClusterIP      10.104.216.189   <none>        8080/TCP                          109s
+```
+
+## Run application (client) requests to ScalarDB or ScalarDL via service resources directly from inside the same Kubernetes cluster
+
+If you deploy your application (client) in the same Kubernetes cluster as ScalarDB or ScalarDL (for example, if you deploy your application [client] on another node group or pool in the same Kubernetes cluster), the application can access ScalarDB or ScalarDL by using Kubernetes service resources. The format of the service resource name (FQDN) is `<HELM_RELEASE_NAME>-envoy.<NAMESPACE>.svc.cluster.local`.
+
+The following are examples of ScalarDB and Scalar DL deployments on the `ns-scalar` namespace:
+
+* **ScalarDB Server**
+    ```console
+    scalardb-envoy.ns-scalar.svc.cluster.local
+    ```
+* **ScalarDL Ledger**
+    ```console
+    scalardl-ledger-envoy.ns-scalar.svc.cluster.local
+    ```
+* **ScalarDL Auditor**
+    ```console
+    scalardl-auditor-envoy.ns-scalar.svc.cluster.local
+    ```
+
+When using the Kubernetes service resource, you must set the above FQDN in the properties file for the application (client) as follows:
+
+* **Client properties file for ScalarDB Server**
+    ```properties
+    scalar.db.contact_points=<HELM_RELEASE_NAME>-envoy.<NAMESPACE>.svc.cluster.local
+    scalar.db.contact_port=60051
+    scalar.db.storage=grpc
+    scalar.db.transaction_manager=grpc
+    ```
+* **Client properties file for ScalarDL Ledger**
+    ```properties
+    scalar.dl.client.server.host=<HELM_RELEASE_NAME>-envoy.<NAMESPACE>.svc.cluster.local
+    scalar.dl.ledger.server.port=50051
+    scalar.dl.ledger.server.privileged_port=50052
+    ```
+* **Client properties file for ScalarDL Ledger with ScalarDL Auditor mode enabled**
+    ```properties
+    # Ledger
+    scalar.dl.client.server.host=<HELM_RELEASE_NAME>-envoy.<NAMESPACE>.svc.cluster.local
+    scalar.dl.ledger.server.port=50051
+    scalar.dl.ledger.server.privileged_port=50052
+    
+    # Auditor
+    scalar.dl.client.auditor.enabled=true
+    scalar.dl.client.auditor.host=<HELM_RELEASE_NAME>-envoy.<NAMESPACE>.svc.cluster.local
+    scalar.dl.auditor.server.port=40051
+    scalar.dl.auditor.server.privileged_port=40052
+    ```
+
+## Expose ScalarDB or ScalarDL via load balancers to run application (client) requests from outside the Kubernetes cluster
```

Comment:
It depends on how we say it, but I think we should use the same wording and sentence structure as the corresponding sentence (`Run application (client) requests to  ...`) above.

Reply:
I see. Following the same structure as the previous heading, how about we use this?:

```suggestion
## Run application (client) requests to ScalarDB or ScalarDL via load balancers from outside the Kubernetes cluster
```

Reply:
Thank you for your suggestion!
It looks good to me. I applied it.